src/components/Dialogs/Dialogs.jsx

```diff
@@ -1,39 +1,57 @@
 import React from "react";
+import { NavLink } from "react-router-dom";
+
+import { ROUTES, I18N } from "../core/constants";
 
 import s from "./Dialogs.module.css";
 
-const Dialogs = (props) => {
+const DialogItem = props => {
+  let path = ROUTES.MESSAGES + "/" + props.id;
```

Comment:
"path" variable is not re-assigned in the current scope (inside the current function), therefore consider defaulting to "const"

Reply:
done, and moved to const file